docs/sources/administration/back-up-grafana/index.md

```diff
@@ -0,0 +1,40 @@
+---
+description: Describes how to back up a locally provisioned Grafana instance.
+keywords:
+  - grafana
+  - backup
+labels:
+  products:
+    - enterprise
+    - oss
+title: Back up Grafana
+weight: 80
+menuTitle: Back up Grafana
+---
+
+# Back up Grafana
+
+This topic explains how to back up a local Grafana deployment, including configuration, plugin data, and the Grafana database.
+
+## Back up the Grafana configuration file
+
+Copy Grafana configuration files that you might have modified in your Grafana deployment to a backup directory.
+
+The Grafana configuration files are located in the following directories:
+
+- Default configuration: `$WORKING_DIR/conf/defaults.ini`
+- Custom configuration: `$WORKING_DIR/conf/custom.ini`
+
+{{% admonition type="note" %}}
+If you installed Grafana using the `deb` or `rpm` packages, then your configuration file is located at
+`/etc/grafana/grafana.ini`. This path is specified in the Grafana `init.d` script using `--config` file parameter.
+{{% /admonition %}}
+
+## Back up plugin data
+
+Installing plugins in Grafana creates a folder for each plugin with its associated files and data. Copy all files and folders recursively from this location to your backup repository.
+
+The Grafana plugin files are located in the following directories:
+
+- Default location for plugins in a binary or source installation: `$WORKING_DIR/data/plugins`
+- Default location for plugins in a `deb` or `rpm` package: `/var/lib/grafana/plugins`. This path is specified in the Grafana init.d script using `--config` file parameter.
```

Comment:
The reason we want there to be a shared content file is so that there's only ever one place to update this content and it doesn't get out of sync between all the places we're using it.
```suggestion
{{< docs/shared lookup="back-up/back-up-grafana.md" source="grafana" version="<GRAFANA VERSION>" >}}
```